### ISSUE TITLE
text alignment no longer works

### ISSUE THREAD

**Issue by frankrolf** [typemytype/drawbot#265]
The alignment of point text seems broken for `'center'` and `'right'` in Version 3.119 (3.119):
```
for alignment_target in ['left', 'center', 'right']:
    newPage(100, 100)
    alignment_point = (width()/2, height()/2)
    with savedState():
        strokeWidth(1)
        stroke(1, 0, 0)
        line((width()/2, 0), (width()/2, height()))
    fs = FormattedString(alignment_target)
    text(fs, alignment_point, align=alignment_target)
```

**Comment by justvanrossum** [comment]
It doesn't seem a new problem, but as far as I can see it's only when using a formatted string. This example works as expected:

```python
for alignment_target in ['left', 'center', 'right']:
    newPage(100, 100)
    alignment_point = (width()/2, height()/2)
    with savedState():
        strokeWidth(1)
        stroke(1, 0, 0)
        line((width()/2, 0), (width()/2, height()))
    text(alignment_target, alignment_point, align=alignment_target)
```

**Comment by justvanrossum** [comment]
I would _think_ that setting the alignment on the fs should be the way to do it, but that doesn't work either:

```python
for alignment_target in ['left', 'center', 'right']:
    newPage(100, 100)
    alignment_point = (width()/2, height()/2)
    with savedState():
        strokeWidth(1)
        stroke(1, 0, 0)
        line((width()/2, 0), (width()/2, height()))
    fs = FormattedString()
    fs.align(alignment_target)
    fs.append(alignment_target)
    text(fs, alignment_point)
    # text(fs, alignment_point, align=alignment_target)
```
However, adding `align` to the `text()` call in _addition to_ the `fs.align()` call seems to work, but that can't be the correct way to do it, I hope.

**Comment by frankrolf** [comment]
This is a new problem. I just worked on a document this morning in the previous version of DrawBot, where the text alignment was handled that way. Now, everything’s off.


from a two-dimensional keyboard

**Comment by justvanrossum** [comment]
> This is a new problem 

You're right, I was looking at an in-between version that was fresher that I thought.

@typemytype, this is the commit that causes the problem:
  https://github.com/typemytype/drawbot/commit/ca29e6f1af1920924d5b7f0f9f9dde672cbc1118

We need a unittest for this case.

**Comment by justvanrossum** [comment]
Actually, it _is_ covered by unittests, and they're failing right now :(

**Comment by typemytype** [comment]
looking into it:

you can always get older version here: http://static.typemytype.com/drawBot/versionHistory/

**Comment by justvanrossum** [comment]
Fixed by reverting #255
